jni/Android.mk

```diff
@@ -90,13 +90,13 @@ $(info $(shell $(NNTRAINER_JNI_ROOT)/prepare_openblas.sh $(NDK_LIBS_OUT)))
 endif #MAKECMDGOALS
 endif #OPENBLAS_ROOT
 
+include $(CLEAR_VARS)
```

Comment:
maybe this should go between L100 and L101?

Reply:
right. 102 is better. fixed.